src/js/containers/awardV2/table/FedAccountTableContainer.jsx

```diff
@@ -0,0 +1,170 @@
+/**
+ * FedAccountTableContainer.jsx
+ * Created by Kwadwo Opoku-Debrah 02/15/2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { isCancel } from 'axios';
+import { uniqueId } from 'lodash';
+
+import * as SearchHelper from 'helpers/searchHelper';
+import * as awardActions from 'redux/actions/awardV2/awardActions';
+
+import BaseFederalAccount from 'models/v2/awards/transactions/BaseFederalAccount';
+
+import FedAccountTable from 'components/award/table/FedAccountTable';
+
+const propTypes = {
+    award: PropTypes.object,
+    category: PropTypes.string
+};
+
+const pageLimit = 15;
+
+export class FedAccountTableContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            inFlight: false,
+            nextPage: false,
+            page: 1,
+            sort: {
+                field: 'piid',
+                direction: 'asc'
+            },
+            tableInstance: `${uniqueId()}`,
+            transactions: []
```

Comment:
In this table, the results aren't transactions, so I would rename this array and `fetchTransactions` to something like `fundingData`